include/GameHandler.hpp

```diff
@@ -3,10 +3,13 @@
 
 #include <string>
 #include "network/DataHandlingService.hpp"
+#include "factory/GameFactory.hpp"
+#include <stdint.h>

```

Comment:
Why is <stdint.h> needed in this file?

Reply:
good catch. I need to check it if it's really necessary

Reply:
ok, this include was unnecessary